streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -382,7 +394,13 @@
                     40 * 1000,
                     atLeast(0),
                     ConfigDef.Importance.MEDIUM,
-                    REQUEST_TIMEOUT_MS_DOC);
+                    REQUEST_TIMEOUT_MS_DOC)
+            .define(PROCESSING_GUARANTEE_CONFIG,
+                    ConfigDef.Type.STRING,
+                    "at_least_once",
+                    in("at_least_once", "exactly_once"),
```

Comment:
I did not do that, to align with ofter config classes. For example, consumer dose not offer "earliest" / "latest" / "none" either. I think consistency is more important at the moment. If we really switch to the builder pattern, this will resolve naturally. WDYT? I am open to change it.